nadlogar/problems/models/meta.py

```diff
@@ -70,7 +72,7 @@ def clean(self):
         if issubclass(Problem, type(self)):
             raise ValidationError("Problems must have a non-trivial generator")
         self.content_type = ContentType.objects.get_for_model(type(self))
-        if hasattr(self, "text") and self.content_type != self.text.content_type:
+        if hasattr(self, "text") and self.text is not None and self.content_type != self.text.content_type:
```

Comment:
Ali to vedno deluje? Najprej pogleda, če ima razred atribut text, potem pa se sklicujemo na ta atribut. Če razred nima texta, je tukaj verjetno error.

Reply:
Sesuti se ne mora, ker najprej pogledamo za prisotnost, nato če ni None in šele nato preberemo `content_type`. V Pythonu `X and Y` deluje tako, da se `Y` sploh ne gre izvajati, če `X` ni resničen. Je pa dobro vprašanje, zakaj bi atribut `self.text` sploh manjkal. Bom še razmislil.

Reply:
Kot gledam, je to zgodovinski ostanek. Bom pokomentiral v PRu od dokumentacije